component/main.jsonnet

```diff
@@ -1,10 +1,36 @@
 // main template for ocp-drain-monitor
+local com = import 'lib/commodore.libjsonnet';
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.ocp_drain_monitor;
 
-// Define outputs below
+local alertlabels = {
```

Comment:
Instead of manually managing common alert labels etc., this should probably use the functions provided by the `alert-patching.libsonnet` library from component openshift4-monitoring, cf. https://hub.syn.tools/openshift4-monitoring/references/alert-patching.html

Reply:
This sounds more like used for patching upstream alerts as far as i can see.

Reply:
I'll create a follow up if we need changes there.